### PR TITLE
Add Debezium Money converter

### PR DESCRIPTION
**Pull request by HelenMel**
A MoneyConverter for debezium connector that convert float money representation to formatted string.

Code source:
https://github.com/moyphilip/MoneyConverter
+ covered with unit tests

### CODE REVIEW

File: src/test/java/io/aiven/kafka/connect/debezium/converters/MoneyConverterTest.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+import io.debezium.spi.converter.CustomConverter;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+public class MoneyConverterTest {
```
Comment (AnatolyPopov) [inline src/test/java/io/aiven/kafka/connect/debezium/converters/MoneyConverterTest.java:35]: What about the case when the column is optional? Should not we also cover it?

File: src/main/java/io/aiven/kafka/connect/debezium/converters/MoneyConverter.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+        if ("money".equals(column.typeName())) {
+            registration.register(moneySchema, data -> {
+                if (data == null) {
+                    if (column.isOptional()) {
+                        return null;
+                    }
+                }
+
+                return String.format("%.2f", data);
```
Comment (AnatolyPopov) [inline src/main/java/io/aiven/kafka/connect/debezium/converters/MoneyConverter.java:50]: Is there any chance that for some reason data will be of any other type than `float`? Is it ok to get `IllegalFormatConversionException` in that case?

File: src/test/java/io/aiven/kafka/connect/debezium/converters/MoneyConverterTest.java [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        transform.configure(prop);
+        assertNull(registration.currFieldSchema);
+        transform.converterFor(new MoneyRelationalColumn(), registration);
+
+        assertEquals(registration.currFieldSchema.schema().name(), "price");
+        assertEquals(registration.currFieldSchema.schema().type(), Schema.Type.STRING);
+    }
+
+    @Test
+    void shouldDoNothingIfColumnIsNotAMoney() {
```
Comment (AnatolyPopov) [inline src/test/java/io/aiven/kafka/connect/debezium/converters/MoneyConverterTest.java:68]:
```suggestion
    void shouldDoNothingIfColumnIsNotMoney() {
```

File: src/test/java/io/aiven/kafka/connect/debezium/converters/MoneyConverterTest.java (hunk 1 above)
Comment (HelenMel) [reply src/test/java/io/aiven/kafka/connect/debezium/converters/MoneyConverterTest.java:35]: Sure!

File: src/main/java/io/aiven/kafka/connect/debezium/converters/MoneyConverter.java (hunk 2 above)
Comment (HelenMel) [reply src/main/java/io/aiven/kafka/connect/debezium/converters/MoneyConverter.java:50]: I assume no. 'MONEY' data type in database should be numeric. So, exception is pretty reasonable here.

**Review by AnatolyPopov — APPROVED** [review]
LGTM

**Pull request merged**